driftdetect/detect_drift.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
This should all be a sub-package of cartography. `cartography.drift` or something similar.